app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -112,34 +130,43 @@
 }
     <div class="govuk-date-input" id="@prefix">
         <div class="govuk-date-input__item">
-            <div class="govuk-form-group">
             @govukInput(Input(
                 id = prefix + "-date-day",
                 name = prefix + "-date-day",
                 value = editSetRecordForm.data.get(prefix + "-date-day"),
                 label = Label(classes = "govuk-label govuk-date-input__label", forAttr = Some("@{prefix}-date-day"), content = Text(messages("edit-set.record.edit.dates.day"))),
-                classes = "govuk-input govuk-date-input__input govuk-input--width-2"))
+                classes = editSetRecordForm.error(prefix+"-date-day") match {
```

Comment:
I'd suggest a more succinct approach, with less duplication:

```
classes = "govuk-input govuk-date-input__input govuk-input--width-2" + editSetRecordForm.error(prefix+"-date-day").map(_ => " govuk-input--error").getOrElse("") 
```

This applies to the other two cases, of course.

Reply:
Done

Reply:
Confirmed.